package.json

```diff
@@ -64,6 +65,7 @@
     "style-loader": "^0.23.1",
     "webpack": "^4.43.0",
     "webpack-cli": "^3.3.11",
-    "webpack-dev-server": "^3.11.0"
+    "webpack-dev-server": "^3.11.0",
+    "husky": "^6.0.0"
```

Comment:
Don't manually add it. Use `npm i husky -D`.

Reply:
@algomaster99 
I actually used the command from [here](https://typicode.github.io/husky/#/). It automatically did this. Same with the above one. So should I position them lexicographically still?

Reply:
Yes